### PR TITLE
Milestone 1:

### PR DESCRIPTION
**Pull request by Bismarck-GM**
# Completed bubble sort:

- Created both Methods/Functions bubble_sort and bubble_sort_by.
- Tested them with Rubocop.
- Added the new Linters with Github Actions.

### CODE REVIEW

**Comment by pelzolga123** [comment]
## Hi Team!  
### You did a good job so far :+1:
Still, there are more changes to make:
* [x] `bubble_sort` method should be able to sort an array of numbers in increasing order. For example:
`bubble_sort([5, 4, 3, 2, 1])` should return `[1, 2, 3, 4, 5]`
* [x] `bubble_sort_by` method should be able to sort an array by accepting a block. For example:
`bubble_sort_by(["hey", "hi", "hello"]){ |left, right| left.length - right.length }` should return `["hi", "hey", "hello"]`

### Please kindly fix the above issues and send another code review request.

If you have any questions or need help, you can message me on slack (@ Olga Pelts). *Happy coding*

File: bubble_sort.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+def bubble_sort(array)
+  iterations = array.length - 1
+  iterations.times do
+    iterations.times do |counter|
+      array[counter], array[counter + 1] = array[counter + 1], array[counter] if array[counter] > array[counter + 1]
+    end
+  end
+end
+
+bubble_sort([6, 5, 4, 3, 2, 1])
```
Comment (pelzolga123) [inline bubble_sort.rb]: No debugging instructions in committed code (as discussed [here](https://questions.microverse.org/t/debugging-instructions-code/346)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/section/debugging_requirement.png) from Pathwright)

**Comment by mikethreels** [comment]
#Milestone 2:
**Changelog**
- deleted debugging code from bubble_sort.rb
- added test.rb
- included return inside method/function.

**Comment by lekegitrepo** [comment]
#### Hi @Bismarck-GM and @mikethreels :100: :1st_place_medal:
##### Well done for implementing the required changes. This project is approved.

### _Reviewed by Temitayo Adeleke._

**Review by lekegitrepo — APPROVED** [review]
#### Status: Approved.

**Pull request merged**